### ISSUE TITLE
The choice of s_det and s_id

### ISSUE THREAD

**Issue by jackie840129** [ifzhang/FairMOT#72]
Hi,
I saw the parameter s_det, s_id in mot.py is set as -1.85 and -1.05, and furthermore, they were fixed along with training (not added in the optimizer).
I wonder how you choose these two values and why I fixed them?
Thank you very much!

**Comment by ifzhang** [comment]
I tried to train them and use the uncertainty loss, but the performance on the validation set do not seem to increase. You can try to add them to optimizer and train them. When training them, the initial values of the two parameters are not very important. I just select them randomly.

**Comment by jackie840129** [comment]
Got it. Thanks a lot!
